File: frontend/src/types/AiProvider.ts
// src/types/AiProvider.ts
export type AiProvider = "openai" | "claude" | "gemini";

export const AI_PROVIDER_LABEL: Record<AiProvider, string> = {
    openai: "GPT",
    claude: "Claude",
    gemini: "Gemini",
};

File: backend/src/main/java/com/lifemetrics/backend/client/AiProvider.java
package com.lifemetrics.backend.client;

/**
 * AI 제공자 타입
 */
public enum AiProvider {
    OPENAI("OpenAI GPT"),
    ANTHROPIC("Anthropic Claude"),
    GEMINI("Google Gemini");
    
    private final String displayName;
    
    AiProvider(String displayName) {
        this.displayName = displayName;
    }
    
    public String getDisplayName() {
        return displayName;
    }
}